Database/Migrations/PetShareDbContextModelSnapshot.cs

```diff
@@ -91,7 +137,30 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("Shelters");
                 });
 
-            modelBuilder.Entity("Database.Entities.PetEnitiy", b =>
+            modelBuilder.Entity("Database.Entities.AnnouncementEntity", b =>
+                {
+                    b.HasOne("Database.Entities.ShelterEntity", "Author")
+                        .WithMany()
+                        .HasForeignKey("AuthorId")
+                        .OnDelete(DeleteBehavior.NoAction)
```

Comment:
That's kind of a nitpick as we don't plan to delete shelters, but we should have `DeleteBehavior.Cascade` here, as announcements cannot really exist without a shelter that created them. The same with `PetEntity`: if a pet doesn't exist anymore, we can't adopt it.

Reply:
Or, if we plan to delete pets (IDK, it kind of makes sense to do so once they are adopted), we could do the option that I forgot the name of, but which enforces that we cannot delete a pet if there are some announcements open for it. This way we would need to do something with those announcements first.

Reply:
Again, that's not really too important.